src/router/routes.ts

```diff
@@ -153,6 +153,68 @@ export const routes: AppRoute[] = [
               },
             ],
           },
+          {
+            path: 'import',
+            name: 'profiles.importLedgerProfile',
+            meta: { protected: false },
+            // @ts-ignore
+            component: () => import('@/views/pages/profiles/import-profile/ImportProfile.vue'),
+            children: [
+              {
+                path: 'inputLedgerAccountInfo',
+                name: 'profiles.importLedgerProfile',
+                meta: {
+                  protected: false,
+                  isLedger: true,
+                  icon: importLedgerStepImage.importLedgerStepImage1,
+                },
+                // @ts-ignore
+                component: () => import('@/views/forms/FormProfileCreation/FormProfileCreation.vue'),
+              },
+              {
+                path: 'importLedger',
+                name: 'accounts.importLedger',
+                meta: {
+                  protected: false,
+                  isLedger: true,
+                  icon: importLedgerStepImage.importLedgerStepImage2,
+                },
+                // @ts-ignore
+                component: () => import('@/views/pages/profiles/import-ledger-profile/ImportLedgerProfile.vue'),
+              },
+            ],
+          },
+          {
+            path: 'import',
+            name: 'profiles.importLedgerProfile',
+            meta: { protected: false },
+            // @ts-ignore
+            component: () => import('@/views/pages/profiles/import-profile/ImportProfile.vue'),
+            children: [
```

Comment:
why double the `/import` top level path instead of merging into one `children` entry?